google-cloud-identity_toolkit-v2/.repo-metadata.json

```diff
@@ -0,0 +1,10 @@
+{
+    "client_documentation": "https://googleapis.dev/ruby/google-cloud-identitytoolkit-v2/latest",
+    "distribution_name": "google-cloud-identitytoolkit-v2",
+    "language": "ruby",
+    "name_pretty": "Google Cloud Identitytoolkit V2 API",
+    "repo": "googleapis/google-cloud-ruby",
+    "requires_billing": true,
+    "ruby-cloud-description": "google-cloud-identitytoolkit-v2 is the official client library for the Google Cloud Identitytoolkit V2 API. Note that google-cloud-identitytoolkit-v2 is a version-specific client library. For most uses, we recommend installing the main client library google-cloud-identitytoolkit instead. See the readme for more details.",
```

Comment:
This description (the first sentence) describes the library, but typically this field is used to describe the service:

```
"ruby-cloud-description": "The BigQuery Connection API allows users to manage BigQuery connections to external data sources. Note that..."
```